File: app/src/main/java/uk/co/amlcurran/social/EmptyCalendarItem.java
package uk.co.amlcurran.social;

public class EmptyCalendarItem implements CalendarItem {
    @Override
    public String title() {
        return "Empty";
    }

    @Override
    public long startDay() {
        return 0;
    }

    @Override
    public boolean isEmpty() {
        return true;
    }
}

File: ios/whatson/Core/EmptyCalendarItem.swift
//
//  EmptyCalendarItem.swift
//  Core
//
//  Created by Alex Curran on 29/06/2019.
//  Copyright © 2019 Alex Curran. All rights reserved.
//

import Foundation

public struct EmptyCalendarItem: CalendarItem {

    public let startTime: Timestamp
    public let endTime: Timestamp
    public let title: String = "Empty"
    public let isEmpty: Bool = true
}
